grails-app/conf/application.yml

```diff
@@ -86,9 +94,23 @@ grails:
                 scriptlets: html
                 taglib: none
                 staticparts: none
-endpoints:
-    jmx:
-        unique-names: true
+---
+headerAndFooter:
+    baseURL: "https://www.ala.org.au/commonui-bs3-2019"
+    version: "2"
+
+skin:
+    layout: "main"
+    orgNameLong: "Atlas of Living Australia"
+    orgNameShort: "ALA"
+    orgSupportEmail: support@ala.org.au
+privacyPolicy: "https://www.ala.org.au/terms-of-use/privacy-policy/"
+siteDefaultLanguage: "en"
+
+security:
+    cas:
+        adminRole: ROLE_ADMIN
+        uriFilterPattern: "/admin/*,/alaAdmin/*"
```

Comment:
Just wondering if the "new" syntax should be used here? E.g. if it was groovy it would be `["/admin/*","/alaAdmin/*"]` so for yaml it might be:
```yaml
    uriFilterPattern:
     - "admin/*"
     - "alaAdmin/*"
```
As per the "Usage" section on https://github.com/AtlasOfLivingAustralia/ala-auth-plugin

Reply:
good idea